Sources/Operations/SwiftMarkOperation.swift

```diff
@@ -13,17 +13,17 @@ public typealias FailureBlock = (SwiftMarkError) -> ()
  /// The `SwiftMarkOperation` class is the abstarct base class for all operation executed in order to convert *CommonMark* texts. Do not subclass or create instances of this class directly. Instead, create instances of one of its concrete subclasses.
  ///
  /// Use the properties of this class to configure the behavior of the operation object before submitting it to an operation queue or executing it directly.
-public class SwiftMarkOperation: NSOperation {
-    private let markdownText: String?
-    private let fileURL: NSURL?
-    private let encoding: UInt
+open class SwiftMarkOperation: Operation {
+    fileprivate let markdownText: String?
```

Comment:
`private` should be preferred for every member of this class since there's only one class definition in this file.